### ISSUE TITLE
Publish DMG in Releases

### ISSUE THREAD

**Issue by khurshid-alam** [sbmpost/AutoRaise#216]
Github direct url in repo has the word "raw" in it. Which is banned in many countries. Hence can not download the dmg easily.

Thanks.

**Comment by sbmpost** [comment]
@khurshid-alam Thanks for reporting. Release 5.4 is now here:
https://github.com/sbmpost/AutoRaise/releases/tag/v5.4

**Comment by khurshid-alam** [comment]
Can you upload the dmg in Asset ?

**Comment by sbmpost** [comment]
@khurshid-alam 

Isn't the dmg in the release?

**Comment by khurshid-alam** [comment]
@sbmpost Nope. It isn’t in the asset. Unless you download the entire source or Git clone.
